### ISSUE TITLE
[500 error]: Cannot set propeties of undefined (setting 'size')

### ISSUE THREAD

**Issue by pothuraju-dev** [nuxt/learn.nuxt.com#129]
Repoduction:
----
1. Go to learn-dev.nuxt.com
2. Got nuxt routing module section by clicking next module buttons
3. Refresh the page after reaching page https://learn-dev.nuxt.com/concepts/routing
4. Again issue coming when come to home page and refresh

Issue:
----
![image](https://github.com/nuxt/learn.nuxt.com/assets/59384635/06d1bb6c-5cb2-4d12-b84c-421dfaab2f92)

**Comment by dhaladitya108** [comment]
@pothuraju-dev can you re-check again? I'm not able to see it now.

**Comment by pothuraju-dev** [comment]
> @pothuraju-dev can you re-check again? I'm not able to see it now.

@dhaladitya108 I just checked it's coming. Follow the steps I mentioned in the issue you will face.

I checked it by cloning the code and figured it out that it's coming from splitpanel npm package used internally in code

**Comment by dhaladitya108** [comment]
> @dhaladitya108 I just checked it's coming. Follow the steps I mentioned in the issue you will face.
> 
> I checked it by cloning the code and figured out that it's coming from splitpanel npm package used internally in code

Yes, @pothuraju-dev! You are right dude. The package doesn't seem to be maintained now.

**Comment by antfu** [comment]
I saw this error occasionally but can't stablely reproduce it. If you spot the culprit, PR is greatly appreciated.

**Comment by pothuraju-dev** [comment]
@rexhent It is coming from third-party package. These errors are inconsistent.

**Comment by rexhent** [comment]
I'm looking into this. I've forked the package https://github.com/rexhent/splitpanes
and am publishing it at splitpanes4. I might refactor it to use composition api and switch away from pug to just normal vue templating syntax, then hopefully I might be able to track down that error more easily. Also more typescript everywhere.

> @rexhent It is coming from third-party package. These errors are inconsistent.

**Comment by rexhent** [comment]
@antfu I'm not exactly sure how patch files work, if it is an issue with the actual splitpanes package if you want you could do it as a pr to https://github.com/rexhent/splitpanes, as splitpanes 3 is seemingly no longer maintained, and the maintainer won't merge any PR's.

**Comment by pothuraju-dev** [comment]
I think this issue was fixed. I checked in the local setup and live, but it didn't occur.

**Comment by rexhent** [comment]
> I think this issue was fixed. I checked in the local setup and live, but it didn't occur.

Pretty sure it's this
https://github.com/nuxt/learn.nuxt.com/commit/7c7a68f5dadcb16493f8a6907f7b3278de104f32

**Comment by amandesai01** [comment]
We at NocoDB faced the same issue with this. This didn't happen when we mounted Splitpanes separately on different pages, but started happening when the pane was auto mounted on route change. The same patch https://github.com/nuxt/learn.nuxt.com/commit/7c7a68f5dadcb16493f8a6907f7b3278de104f32 fixed the issue.

Thanks @antfu for the patch.
